pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -508,8 +508,11 @@ def _create_server_instance(
               metavar='INT',
               # defaulted in code
               envvar=PYWBEMCLI_TIMEOUT_ENVVAR,
-              help=u'Client-side timeout in seconds for operations with the '
-                   u'WBEM server. '
+              help=u'Client-side timeout (seconds) on data read for '
+                   u'operations with the WBEM server. This integer is the '
+                   u'timeout for a single server request. Pywbem retries '
+                   u'multiple times so the delay for read timeout '
+                   u'failure may be multiple times this value.'
```

Comment:
Since the number of retries is not configurable, we could state it in the help text, by taking it from the constant that defines it.

Reply:
I did not do that here because the variables are internal in pywbem._cim_operations.py.   I think we should make them more public than they are in pywbem itself but will make the the change here for now.

Reply:
Fixed to show the value of the read timeout.  Note that we do not have anyplace to show any of the other timeout values (connect, etc.)